Apps/WebClient/src/ClientApp/src/components/timeline/errorCard.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+    <div>
+        <b-alert show="show" variant="danger" dismissible class="no-print">
+            <h4>{{ getCodeText() }} {{ title }}</h4>
+            <span>{{ description }}</span>
+        </b-alert>
+    </div>
+</template>
+
+<script lang="ts">
+import Vue from "vue";
+import { Component, Prop } from "vue-property-decorator";
+
+@Component
+export default class ErrorCardComponent extends Vue {
+    @Prop() title: string;
+    @Prop() description: string;
+    @Prop() code: string;
+    @Prop() show: boolean;
+
+    private getCodeText(): string {
+        if (!this.code) {
```

Comment:
how about a shorter/1-line version of this as follows?
`return !this.code ? "" : this.code + ": ";`

Reply:
i would make it the other way around ```return this.code ? this.code + ": ": "" ;

Reply:
Will do.

Reply:
> i would make it the other way around `return this.code ? this.code + ": ": "" ;`
Way better with this :)